src/main/java/net/javadiscord/javabot/systems/help/dao/HelpAccountRepository.java

```diff
@@ -0,0 +1,90 @@
+package net.javadiscord.javabot.systems.help.dao;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import net.javadiscord.javabot.systems.help.model.HelpAccount;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Optional;
+
+/**
+ * Dao class that represents the HELP_ACCOUNT SQL Table.
+ */
+@Slf4j
+@RequiredArgsConstructor
+public class HelpAccountRepository {
+	private final Connection con;
+
+	/**
+	 * Inserts a new {@link HelpAccount}.
+	 *
+	 * @param account The account that should be inserted.
+	 * @throws SQLException If an error occurs.
+	 */
+	public void insert(HelpAccount account) throws SQLException {
+		try (PreparedStatement s = con.prepareStatement("INSERT INTO help_account (user_id, experience) VALUES ( ?, ? )")) {
```

Comment:
Do we really need that table? Couldn't we just use the sum of all experiences of that user in the transaction table?